tests/cmdline/commands/test_process.py

```diff
@@ -490,3 +393,84 @@ def test_multiple_processes(self):
         self.assertIn('No callers found', get_result_lines(result)[0])
         self.assertIn(str(self.node_root.pk), get_result_lines(result)[1])
         self.assertIn(str(self.node_root.pk), get_result_lines(result)[2])
+
+
+@pytest.mark.skip(reason='fails to complete randomly (see issue #4731)')
+@pytest.mark.requires_rmq
+@pytest.mark.usefixtures('with_daemon')
+@pytest.mark.usefixtures('clear_database_before_test')
```

Comment:
You can use a single mark statement to inject fixtures
```suggestion
@pytest.mark.usefixtures('with_daemon', 'clear_database_before_test')
```